cloudmanager/projects.go

```diff
@@ -37,8 +37,8 @@ type ProjectsService interface {
 	Delete(context.Context, string) (*atlas.Response, error)
 }
 
-//ProjectsServiceOp handles communication with the Projects related methos of the
-//MongoDB Atlas API
+// ProjectsServiceOp handles communication with the Projects related methos of the
+// MongoDB Cloud Manager API
```

Comment:
np: May have been the intention to add `See more: https://docs.cloudmanager.mongodb.com/reference/api/groups/` here as opposed to replacing it above? If not, maybe we could add that here anyway

Reply:
I haven't added any see more, I corrected the one we used to have, as those were copy/paste errors from when this was first done